src/dataExplorer/components/BucketSelector.tsx

```diff
@@ -15,9 +15,11 @@ import SelectorTitle from 'src/dataExplorer/components/SelectorTitle'
 import {ScriptQueryBuilderContext} from 'src/dataExplorer/context/scriptQueryBuilder'
 import {BucketContext} from 'src/shared/contexts/buckets'
 import {event} from 'src/cloud/utils/reporting'
+import {PersistanceContext} from 'src/dataExplorer/context/persistance'
```

Comment:
Thank you with your fresh eyes! I didn't pay attention to it since it was introduced 😅 Seems like there are a lot to refactor, so it worths another PR. I created an issue for it https://github.com/influxdata/ui/issues/6649

Reply:
This is dumb and nit-picky of me, I just flagged it in case it could be resolved with another edit. 😅 It has zero impact on the behavior here.